com.unity.multiplayer.mlapi/Editor/CodeGen/ILPostProcessCompiledAssembly.cs

```diff
@@ -0,0 +1,42 @@
+#if !UNITY_2020_2_OR_NEWER || UNITY_2021_1_OR_NEWER
+using System.IO;
+using Unity.CompilationPipeline.Common.ILPostProcessing;
+
+namespace MLAPI.Editor.CodeGen
+{
+    internal class ILPostProcessCompiledAssembly : ICompiledAssembly
+    {
+        private readonly string m_AssemblyFilename;
+        private readonly string m_OutputPath;
+        private InMemoryAssembly m_InMemoryAssembly;
+
+        public ILPostProcessCompiledAssembly(string asmName, string[] refs, string[] defines, string outputPath)
+        {
+            m_AssemblyFilename = asmName;
+            Name = Path.GetFileNameWithoutExtension(m_AssemblyFilename);
+            References = refs;
+            Defines = defines;
+            m_OutputPath = outputPath;
+        }
+
+        public string Name { get; }
```

Comment:
----------------------------------------------------------------
        public string Name { get; }
        public string[] References { get; }
        public string[] Defines { get; }
        ========================= (C# standard?)
        public string name { get; }
        public string[] references { get; }
        public string[] defines { get; }
----------------------------------------------------------------
        
        _Just calling out the public fields that might need lower case?_